csharp/ql/lib/semmle/code/csharp/commons/StructuralComparison.qll

```diff
@@ -284,12 +277,6 @@ module Internal {
      */
     abstract predicate candidate(ControlFlowElement x, ControlFlowElement y);
 
-    pragma[inline]
```

Comment:
Yes, great to get rid of this duplication.
I hope we can get completely rid of the configuration classes.